src/test/store/receive-profile.test.js

```diff
@@ -1661,6 +1686,73 @@ describe('actions/receive-profile', function() {
       const nodeData = callTree.getNodeData(firstChild);
       expect(nodeData.self).toBe(4);
     });
+
+    it("doesn't includes screenshot track of both profiles", async function() {
+      const store = blankStore();
+      const { resultProfile } = await setup(getSomeProfiles(), {
+        url1: 'https://fakeurl.com/public/fakehash1/?thread=0&v=3',
+        url2: 'https://fakeurl.com/public/fakehash1/?thread=0&v=3',
+      });
+
+      store.dispatch(viewProfile(resultProfile));
+      expect(getHumanReadableTracks(store.getState())).toEqual([
+        'show [thread Empty default] SELECTED',
+        'show [thread Empty default]',
+        'show [thread Diff between 1 and 2 comparison]',
+      ]);
+    });
+
+    it('includes screenshot track of both profiles', async function() {
+      const store = blankStore();
+      //Get profiles with one screenshot track
+      const profile1 = getProfileWithMarkers([
+        [
+          'CompositorScreenshot',
+          0,
+          null,
+          {
+            type: 'CompositorScreenshot',
+            url: 0, // Some arbitrary string.
+            windowID: 0,
+            windowWidth: 300,
+            windowHeight: 150,
+          },
+        ],
+      ]);
+      const profile2 = getProfileWithMarkers([
+        [
+          'CompositorScreenshot',
+          0,
+          null,
+          {
+            type: 'CompositorScreenshot',
+            url: 0, // Some arbitrary string.
+            windowID: 1,
+            windowWidth: 300,
+            windowHeight: 150,
+          },
+        ],
+      ]);
+      const { resultProfile } = await setup(
+        {
+          profile1: profile1,
+          profile2: profile2,
+        },
+        {
+          url1: 'https://fakeurl.com/public/fakehash1/?thread=0&v=3',
+          url2: 'https://fakeurl.com/public/fakehash1/?thread=0&v=3',
+        }
+      );
+
+      store.dispatch(viewProfile(resultProfile));
+      expect(getHumanReadableTracks(store.getState())).toEqual([
+        'show [screenshots]',
+        'show [screenshots]',
+        'show [thread Empty default] SELECTED',
+        'show [thread Empty default]',
+        'show [thread Diff between 1 and 2 comparison]',
+      ]);
+    });
```

Comment:
Added these, looking forward to your opinion. @canova

Reply:
Looks pretty good to me, thanks!